router/proxy/export.go

```diff
@@ -0,0 +1,3 @@
+package proxy
+
+var StickyCookie = stickyCookie
```

Comment:
Should this be a constant instead of a var? Was this intentionally left in a separate file like this?

Seems weird that this can be globally overridden.